### PR TITLE
Feature/preliminary testing

### LINKED ISSUE #28: Function for retrieving relevant clothing data for survey

### LINKED ISSUE #37: Refactoring view in state to be dynamic
As of now, it has a fixed number of pages to navigate through. We could use `Object.keys` on the object we receive from our network request possibly.

### LINKED ISSUE #45: 400 error code from recent rebase
Will get more details tomorrow, it started happening when we rebased the graphql branch into the `feature/preliminary-testing` branch.

### PR DESCRIPTION
**Pull request by AurumValian**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
This PR adds a couple of tests to the Survey component. It also creates cards for each image that are selected when clicked. Currently it's locked on a single selection.

After the most recent rebase, we had a 400 error code (bad request). We will solve tomorrow.

###### Screenshots (if appropriate):
Screen Shot 2020-10-15 at 7 55 25 PM

### CODE REVIEW

**Review by amykarnaze — APPROVED** [review]
Nice driving, Ben. Code is coming along!

**Review by nickhartdev — APPROVED** [review]
Appreciate you driving for this Ben! Excited to see this all come together

File: code/web/src/modules/survey/api/actions.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const setSurveyType = crateId => (
           crateId
         })
       })
-      .catch(error => console.error(error))
+      .catch(error => console.error('There\'s an error'))
```
Comment (nickhartdev) [inline code/web/src/modules/survey/api/actions.js:35]: This'll get tweaked, was originally implemented for testing

Comment (ejdelsztejn) [reply code/web/src/modules/survey/api/actions.js:35]: Sounds good

**Comment by ejdelsztejn** [comment]
Looks good! Great work, Front-End team!

**Review by ejdelsztejn — APPROVED** [review]

**Pull request merged**